modules/core/keeper/msg_server.go

```diff
@@ -193,32 +193,24 @@ func (k *Keeper) ConnectionOpenConfirm(goCtx context.Context, msg *connectiontyp
 func (k *Keeper) ChannelOpenInit(goCtx context.Context, msg *channeltypes.MsgChannelOpenInit) (*channeltypes.MsgChannelOpenInitResponse, error) {
```

Comment:
This is probably the file with most changes. They can be quite easily summarized as follows:

- For each method that uses capability:
   - Remove usages of `LookupModule...` and replace with direct call to the (now changed) `Route` method.
   - Since we do not get the capability anymore from the function above, remove it as an argument to underlying functions